app/assets/stylesheets/modules/show_playdates.scss

```diff
@@ -0,0 +1,27 @@
+html, body {
+  height: 100%;
+  margin: 0;
+}
+
+#map {
+  width: 450px;
+  height: 550px;
+  border-width: 7px;
+  box-shadow: 2px 2px 10px #B1B1B1;
+  border-style: solid;
+
+}
+
+
+
+#map-container {
+   height: 400px;
+   width: 300px;
+   border-radius: 16px 16px;
+   border-color: #fff;
+   margin-top: 25px;
+ }
+ #map-canvas {
```

Comment:
Avoid using id selectors<br>Line should be indented 0 spaces, but was indented 1 spaces